code/modules/reagents/machinery/chem_dispenser.dm

```diff
@@ -353,11 +352,14 @@
 	circuit = /obj/item/weapon/circuitboard/chemical_dispenser/industrial
 	level0 = list(
 		"acetone","aluminum","ammonia",
-		"copper","ethanol","hydrazine",
-		"iron","radium","sacid",
-		"hclacid","silicon","tungsten"
+		"copper","ethanol","hclacid",
+		"hydrazine","iron","phosphorus",
+		"potassium", "radium","sacid",
+		"silicon","sulfur","tungsten"
 	)
 
 	level1 = list("oil", "cryptobiolin", "sterilizine")
+	level2 = list("foaming_agent", "plasticide", "instant_ice")
```

Comment:
```suggestion
	level2 = list("foaming_agent", "plasticide", "instant_ice", "toxin", "sodiumchloride")
```
you likely didnt know, but ye this needs to be added to the list for consistancy

Reply:
I figured it's an industrial dispenser, for industrial-oriented work (hence plasticide, foaming agent, etc), how does salt fit on it?